trouper.py

```diff
@@ -0,0 +1,14 @@
+from math import ceil
+
+
+class Number(int):
+    """
+    Saves a number and shows the number's divisors.
+    """
+
```

Comment:
Missing slice support and contains support

Reply:
Slice is supported by the method __getitem__ with argument of type 'slice'.
The __getitem__ method sends each argument to int.__getitem__ and it supports argument of type 'slice'. So slice is supported.